web/components/inbox/modals/create-edit-modal/issue-description.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC, RefObject } from "react";
+import { observer } from "mobx-react";
+import { EditorRefApi } from "@plane/rich-text-editor";
+import { TIssue } from "@plane/types";
+import { Loader } from "@plane/ui";
+// components
+import { RichTextEditor } from "@/components/editor/rich-text-editor/rich-text-editor";
+// hooks
+import { useProjectInbox } from "@/hooks/store";
+
+type TInboxIssueDescription = {
+  workspaceSlug: string;
+  projectId: string;
+  workspaceId: string;
+  data: Partial<TIssue>;
+  handleData: (issueKey: keyof Partial<TIssue>, issueValue: Partial<TIssue>[keyof Partial<TIssue>]) => void;
+  editorRef: RefObject<EditorRefApi>;
+};
+
+// TODO: have to implement GPT Assistance
+export const InboxIssueDescription: FC<TInboxIssueDescription> = observer((props) => {
+  const { workspaceSlug, projectId, workspaceId, data, handleData, editorRef } = props;
+  // hooks
+  const { isLoading } = useProjectInbox();
```

Comment:
isLoading should be a boolean, please use appropriate names in the future. if its a string type use something like a loader in the future